Test/CoreSDK.Test/Net40/Core.Net40.Tests.csproj

```diff
@@ -24,6 +24,9 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' " />
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' " />
   <ItemGroup>
+    <Reference Include="KellermanSoftware.Compare-NET-Objects, Version=3.8.0.0, Culture=neutral, PublicKeyToken=d970ace04cc85217, processorArchitecture=MSIL">
```

Comment:
@cijothomas @Dmitry-Matveev According to NuGet https://github.com/GregFinzer/Compare-Net-Objects/wiki/Licensing this component is licensed under MS public license (Ms-PL) so I do not think any extra approval is necessary... but maybe we need to "register the use" or something.